site/docs/spec.md

```diff
@@ -989,6 +1007,26 @@ Table metadata is serialized as a JSON object according to the following table.
 |**`sort-orders`**|`JSON sort orders (list of sort field object)`|`See above`|
 |**`default-sort-order-id`**|`JSON int`|`0`|
 
```

Comment:
Nit: looks like the convention is to have two blank lines between sections, so add one here?